erpnext/setup/doctype/vehicle/vehicle.json

```diff
@@ -597,20 +586,7 @@
    "in_list_view": 1,
    "in_standard_filter": 1,
    "label": "Fuel Type",
-   "length": 0,
-   "no_copy": 0,
-   "options": "Petrol\nDiesel\nNatural Gas\nElectric",
-   "permlevel": 0,
-   "precision": "",
-   "print_hide": 0,
-   "print_hide_if_no_value": 0,
-   "read_only": 0,
-   "remember_last_selected_value": 0,
-   "report_hide": 0,
-   "reqd": 1,
-   "search_index": 0,
-   "set_only_once": 0,
-   "unique": 0
+   "options": "\nPetrol\nDiesel\nNatural Gas\nElectric\nMuscle"
```

Comment:
Unsure about adding “muscle”. This seems more funny than relevant.
“Hydrogen” and “Hybrid” is missing, too.
E-Bikes might be considered hybrid, too.
Maybe just add “Other”?

Reply:
Yes, you're right. This doctype was so entrenched on cars that it's still far away from a general understanding of what a vehicle may be. So this became inadvertently a bit funny. 😁

I'll add these suggestions, though.

Reply:
I agree. There‘s so much more to do until being useful for cars, bikes and other vehicles alike. But your PR will at least remove car-focussed requirements that are currently showstoppers for bikes. 👍

Reply:
https://github.com/frappe/erpnext/pull/37321/commits/e17a6aa9e0e2b8272c68f068e525733c5fd367f8

Reply:
Ok, this now has gotten a little rewiring based on my experiences of running a VRP solver in ERPNext in production now for a couple of months. This is an iteration and if accepted, I may consider to release the VRP app in the future.

Reply:
Now it's coming together, yes!
I was kind of wondering what it is all about, but with the VRP solver in mind, it completely makes sense. 👍